index.js

```diff
@@ -53,6 +53,7 @@ var config = {
 	animateInParallel: false,
 	appIcon: null,
 	pathToModule: '',
+	autoCleanup: true, // Auto cleanup
```

Comment:
I don't see why not, by default if you don't cleanup the nw.exe processes will just keep running until you restart your system and that's a very very bad design imo.